cg_openmm/simulation/rep_exch.py

```diff
@@ -146,18 +142,16 @@ def run_replica_exchange(
     topology,
     system,
     positions,
-    temperature_list=None,
-    simulation_time_step=None,
     total_simulation_time=1.0 * unit.picosecond,
-    output_data="output.nc",
-    print_frequency=100,
-    collision_rate=5,
-    verbose_simulation=False,
+    simulation_time_step=None,
+    temperature_list=None,
+    friction=1.0 / unit.picosecond,
+    minimize=True,
     exchange_frequency=1000,
-    test_time_step=False,
-    output_directory=None,
-    minimize=True
-    ):
+    print_frequency=100,
```

Comment:
Looks like print_frequency is not used in the openmmtools replica exchange? As far as I can tell, energies are only written after each exchange iteration, and we can only change when the checkpoint .nc files are written.

Reply:
Yeah, it's not right now.  I'm cutting it out, and we can perhaps add later if we decide to.